lib/serverless-tools/notifier/message.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+require "octokit"
+
+module ServerlessTools
+  module Notifier
+    class Message
```

Comment:
This class seems to have more than one responsibility. It fetches data from Github, knows how to construct the slack names of people from the GH author, formats a MD message. Would it be overkill to split this up? Or rethink how it's constructed. Or, not?

Also the name is quite generic `Message`, but the implementation is quite specific, and possibly more like a `DeploymentStatusMessage` - Could we have other messages, and is that worth thinking about?

Reply:
After a chat with Paul we decided I can look into some refactoring of this class as a follow-up PR (e.g. extract workflow run information into a separate class, which would greatly simplify the message)